src/main/resources/com/cloudbees/hudson/plugins/folder/AbstractFolder/view-index-top_ja.properties

```diff
@@ -21,3 +21,4 @@
 # THE SOFTWARE.
 
 Folder\ name=\u30d5\u30a9\u30eb\u30c0\u540d
+Full\ folder\ name=\u5b8c\u5168\u306a\u30d5\u30a9\u30eb\u30c0\u540d 
```

Comment:
Uh, do you speak Japanese fluently? As a general rule, no one other than translators should touch localized files except perhaps to delete keys deleted from the base file, or move keys moved in base files.